### DISCUSSION TITLE
What is `INTERNAL_EDGE_FUNCTION_UNHANDLED_ERROR`?

### DISCUSSION THREAD

**Discussion by michaelaflores** [vercel/next.js#43547] [Help]
I was looking for this error in https://vercel.com/docs/errors but I don't see it. Is there a good place to learn more about its causes?

**Comment by someguy9** [comment]
Same issue here

**Comment by longlostnick** [comment]
Same here.

**Comment by k9ordon** [comment]
Same here.

**Comment by mmachatschek** [comment]
Seeing elevated error rates with this above error. Unfortunately no response by Vercel team. Very sad!
